doc/cascadia/SettingsSchema.md

```diff
@@ -90,69 +90,57 @@ Properties listed below are specific to each custom key binding.
-- newTabProfile0
-- newTabProfile1
-- newTabProfile2
-- newTabProfile3
-- newTabProfile4
-- newTabProfile5
-- newTabProfile6
-- newTabProfile7
-- newTabProfile8
-- closeWindow
-- closeTab
-- closePane
-- switchToTab
-- nextTab
-- prevTab
-- increaseFontSize
-- decreaseFontSize
-- resetFontSize
-- scrollUp
-- scrollDown
-- scrollUpPage
-- scrollDownPage
-- switchToTab0
-- switchToTab1
-- switchToTab2
-- switchToTab3
-- switchToTab4
-- switchToTab5
-- switchToTab6
-- switchToTab7
-- switchToTab8
-- openSettings
-- splitPane
-- resizePaneLeft
-- resizePaneRight
-- resizePaneUp
-- resizePaneDown
-- moveFocusLeft
-- moveFocusRight
-- moveFocusUp
-- moveFocusDown
-- toggleFullscreen
-- find
-
-## Example Keys
-- ctrl+1
-- ctrl+plus
-- alt+-
-- shift+numpad_1
-- ctrL+shift+numpad_plus
-- ctrl+pgdn
-- ctrl+alt+shift+pgup
+| `action` | Optional | String | Adds additional functionality to certain commands. |
+
+### Implemented Commands and Actions
+
+Commands listed below are per the implementation in [`src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp`](https://github.com/microsoft/terminal/blob/master/src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp).
+
+Keybindings can be structured in the following manners:
+<br>
+`{ "command": "commandName", "keys": [ "modifiers+keys" ] }`
+<br>
+`{ "command": { "action": "commandName", "actionName": "actionArguments" }, "keys": ["modifiers+keys"] }`
+
+| Command | Action | Action Arguments |
+| ------- | ------ | ---------------- |
+| closePane | | |
+| closeTab | | |
+| closeWindow | | |
+| copy | `trimWhitespace` | boolean |
+| decreaseFontSize | `delta` | integer |
+| duplicateTab | | |
+| find | | |
+| increaseFontSize | `delta` | integer |
+| moveFocus | `direction` | `left`, `right`, `up`, `down` |
+| newTab | `commandLine`<br>`startingDirectory`<br>`tabTitle`<br>`index`<br>`profile` | string<br>string<br>string<br>integer<br>string |
+| nextTab | | |
+| openNewTabDropdown | | |
+| openSettings | | |
+| paste | | |
+| prevTab | | |
+| resetFontSize | | |
+| resizePane | `direction` | `left`, `right`, `up`, `down` |
+| scrollDown | | |
+| scrollUp | | |
+| scrollUpPage | | |
+| scrollDownPage | | |
+| splitPane | `split`<br>`commandLine`<br>`startingDirectory`<br>`tabTitle`<br>`index`<br>`profile` | `vertical`, `horizontal`, `auto`<br>string<br>string<br>string<br>integer<br>string |
+| switchToTab | `index` | integer |
+| toggleFullscreen | | |
+| unbound | | |
+
+### Accepted Modifiers and Keys
+
+#### Modifiers
+`Ctrl+`, `Shift+`, `Alt+`
+
+#### Keys
+- `f1 - f24`, `a-z`, `0-9`
+- ``` ` ```, `-`, `=`, `[`, `]`, `\`, `;`, `'`, `,`, `.`, `/`
+- `down`, `left`, `right`, `up`, `pagedown`, `pageup`, `pgdn`, `pgup`, `end`, `home`, `plus`
+- `tab`, `enter`, `esc`, `escape`, `space`, `backspace`, `delete`, `insert`
+- `numpad_0-numpad_9`, `numpad0-numpad9`, `numpad_add`, `numpad_plus`, `numpad_decimal`, `numpad_period`, `numpad_divide`, `numpad_minus`, `numpad_subtract`, `numpad_multiply`
```

Comment:
Personal preference: preface the keys with text. Otherwise I'm just staring at a bunch of keys.

Maybe make it a table like this?

| | |
|--|--|
| Function Keys | `f1`, `f2`, etc... |
| Arrow Keys |  `down`, `left`, `right`, `up` |
| Numpad Keys | `numpad_0-numpad_9`, `numpad0-numpad9`, `numpad_add`, `numpad_plus`, `numpad_decimal`, `numpad_period`, `numpad_divide`, `numpad_minus`, `numpad_subtract`, `numpad_multiply`|